### ISSUE TITLE
timeupdate event off when seek backwards fired in Chrome

### ISSUE THREAD

**Issue by petarsubotic** [videojs/video.js#3977]
## Description
timeupdate event and currentTime value respectively stop updating on one click backwards seek in Chrome.
e.g. http://videojs.com/advanced/

## Steps to reproduce
Exact steps necessary to reproduce the issue.
1. Play the video 
2. Observe timeupdate event count as well as currentTime value
2. When the video reaches 10 seconds, without pausing, click on the timeline to second 2.
3. Observe timeupdate event count as well as currentTime value

### Expected
timeupdate event should continue to fire continually
currentTime should be updated respectively

### Actual
timeupdate event is off until the player timeline reaches 10s
currentTime is not updating respectively to timeupdate

### Additional information
1. currentTime appears to update to accurate value for a split second during the click event but instantly reverses to the original point at 10 seconds
2. on double click, both the timeupdate and currentTime revive as they should
3. issue seems to be isolated to Google Chrome

#### videojs
5.16.0
#### browsers
Google Chrome
#### OSes
Windows

**Comment by davidjamesherzog** [comment]
I updated the example with observing the `timeupdate` event - https://jsfiddle.net/cnpza7uj/4/.  This helps display the described behavior in this issue.  I just updated to Chrome v56 and I no longer see this behavior.  I have another machine with Chrome v55 and I still see the behavior.  I believe this to be more of a Chrome issue than a video.js issue.  I don't know how many versions back in Chrome this problem existed.
